<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#btnIgual, #btnMais, #btnMenos, #btnVezes, #btnDivisao{
	color: #000;
	background-color: orange;
	margin: 1px;
}

#btnPorcentagem, #btnParenteses, #btnApagarTdo {
	background-color: black;
	color: white;
	margin: 1px;
}
#centralwidget {
	background: #ddd;
}

#btnApagar {
	background-color: red;
	margin: 4px;
	font-size: 2rem;
	border-radius: 5px;
	color: white;
}

.QLabel {
	height: 20px;
}

.QPushButton::hover {
	background-color: #fff;
	border-radius: 10px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnApagarTdo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
    <property name="shortcut">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnParenteses">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>()</string>
    </property>
    <property name="shortcut">
     <string>(, )</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPorcentagem">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDivisao">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
    <property name="shortcut">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="shortcut">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
    <property name="shortcut">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="shortcut">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="shortcut">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="shortcut">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="shortcut">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="shortcut">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenos">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="shortcut">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVezes">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="shortcut">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMaisMenos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMais">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="shortcut">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="shortcut">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZero">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="shortcut">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIgual">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVirgu">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
    <property name="shortcut">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApagar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>70</width>
      <height>49</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnApagar::hover {
	background-color: #ff3b19;
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>โซ</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="display_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
